cmd/nginx-auth/mkdeb.sh

```diff
@@ -2,30 +2,31 @@
 
 set -e
 
-CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -o tailscale.nginx-auth .
+for ARCH in amd64 arm64; do
+    CGO_ENABLED=0 GOARCH=${ARCH} GOOS=linux go build -o tailscale.nginx-auth .
+    VERSION=0.1.3
```

Comment:
I suppose this could remain at the top level since it's shared. YMMV.

Reply:
Done.